bin/strings.json

```diff
@@ -141,6 +141,7 @@
         "COMPILE_ARG_TOOLCHAIN" : "Specify the NDK_TOOLCHAIN of ndk-build.",
         "COMPILE_ARG_CPPFLAGS" : "Specify the APP_CPPFLAGS of ndk-build.",
         "COMPILE_ARG_NO_APK" : "Not generate the apk file.",
+        "COMPILE_ARG_NO_SIGN" : "Generate unsigned apk even when release mode.",
```

Comment:
Support to generate unsigned apk, it's meanful when `gen-libs`. Another benefit, developer can test release compile with no sign input.